sdk/highlight-go/tracer.go

```diff
@@ -79,12 +73,6 @@ func (t Tracer) InterceptField(ctx context.Context, next graphql.Resolver) (inte
 // InterceptResponse instruments timing, payload size, and error information
 // of the response handler. The metric is grouped by the corresponding operation name.
 func (t Tracer) InterceptResponse(ctx context.Context, next graphql.ResponseHandler) *graphql.Response {
-	// if we don't have a highlight session in the context, no point trying to
-	// instrument since we won't be able to store the metric
-	if _, _, err := validateRequest(ctx); err != nil {
```

Comment:
this comment is out of date, from back when we were using this code path to record session metrics. traces can be recorded for a project without a session / request id.